contrib/cirrus/lib.sh

```diff
@@ -294,8 +294,8 @@ setup_rootless() {
 
     cd $GOSRC
     # Guarantee independence from specific values
-    ROOTLESS_UID=$[RANDOM+1000]
-    ROOTLESS_GID=$[RANDOM+1000]
+    ROOTLESS_UID=$((RANDOM+1000))
+    ROOTLESS_GID=$((RANDOM+1000))
```

Comment:
I haven't researched this much, but we have to be careful here b/c bash is potentially very old under some circumstances.  Are you certain the `$((...))` way functions in vanilla (not updated) Ubuntu 18?